docs/endaq/quickstart.rst

```diff
@@ -80,3 +82,115 @@ information about the device and how it was configured.
+                     "message: {dev.command.status[1]}")
+               if dev.command.status[1] == endaq.device.response_codes.DeviceStatusCode.RECORDING or \
+                     dev.command.status[1] == endaq.device.response_codes.DeviceStatusCode.IDLE_UNMOUNTED:
+                  print(
+                     f"Device is recording. Status code: {dev.command.status[1]}")
+
+         except Exception as e:
+               print(f"Got error {e}")
+
+         time.sleep(1)
+
+
+   # Accessing basic properties about the device:
+   # This is done by calling different properties of our instance of
+   # the Recorder class called "dev".
+   print("PROPERTIES")
+   print("Device Name:", dev.name)
+   print("Device Serial Number:", dev.serial)
+   print("Device Hardware Version:", dev.hardwareVersion)
+   print("Device Firmware Version:", dev.firmwareVersion)
+   print("Device Channels:", dev.channels)
+   # To view all possible properties to access, look at the "The Recorder Class"
+   # section of the documentation.
+
+   # Some basic configurations:
+   print("\nCONFIGURATION")
+   print("Device ready for configuration:", dev.config.available)
+   dev.config.enableChannel(dev.channels[80], False)  # Disable Ch 80
+   dev.config.enableChannel(dev.channels[80])
+   dev.config.setSampleRate(dev.channels[80], 1000)  # Set Ch 80 SR to 1000 Hz
+   print("New Channel 80 Sample Rate:",
+         dev.config.getSampleRate(dev.channels[80]))
+   dev.config.revert()  # Revert to default original configuration
+   print("Original Channel 80 Sample Rate:",
+         dev.config.getSampleRate(dev.channels[80]))
+   print("Settable triggers for Channel 80:",
+         dev.config.getTrigger(dev.channels[80]))
+   print("Settable triggers for all channels:", dev.config.getTriggers())
+   dev.config.setTrigger(dev.channels[80], high=20)
+   dev.config.applyConfig()
+   # To view all possible configurations, look at the "Configuration and Control"
+   # section of the documentation.
+
+   # Some basic commands:
+   # Note: Commands can only be sent to a physical enDAQ device;
+   # They will not work on Virtual Devices (imported .IDE files).
+   print("\nCOMMANDS")
+   print("Device ready for commands:", dev.command.available)
+   print("Device Battery Status:", dev.command.getBatteryStatus())
+   print("Device Clock Drift:", dev.command.getClockDrift())
+   print("Device Time:", dev.command.getTime())
+   dev.command.setTime()  # Set the device time to the computer's time
+   dev.command.blink()  # Blink the device's LEDs
+   # 'blink()' is helpful for identifying a specific device when multiple devices
+   # are plugged in to one computer
+   commandWait(10)  # Defined above! Waits and prints useful information.
+   dev.command.startRecording()
+   commandWait(10)
+   print("Recording stopped:", dev.command.stopRecording())
+   # To view all possible commands, look at the "Configuration and Control" section
+   # of the documentation.
+
+   # Virtual Devices:
```

Comment:
I don't think the virtual device should be in this example. Most users won't have the `test.ide` file, and there's already a code snippet showing how to instantiate a virtual device.